extension/src/experiments/model/index.ts

```diff
@@ -108,6 +114,19 @@ export class ExperimentsModel extends ModelWithPersistence {
     this.setColoredStatus()
   }
 
+  public toggleStars(ids: string[]) {
+    for (const id of ids) {
+      const current = this.starredExperiments[id]
+
+      if (current) {
+        delete this.starredExperiments[id]
+      } else {
+        this.starredExperiments[id] = STARRED
```

Comment:
[Q] Do you think there will be some in-between state in the future? Why not start with a boolean value?